docs/reference/commandline/plugin_rm.md

```diff
@@ -12,20 +12,22 @@ parent = "smn_cli"
 # plugin rm (experimental)
 
 ```markdown
-Usage:  docker plugin rm PLUGIN
+Usage:  docker plugin rm [OPTIONS] PLUGIN [PLUGIN...]
 
 Remove a plugin
 
 Aliases:
   rm, remove
 
 Options:
-      --help   Print usage
+      -f, --force  Force the removal of an active plugin
```

Comment:
Small formatting nit; `--force` should align with `--help` (that's the format Cobra uses);

    -f, --force .....
        --help ....